libopenage/versions/versions.cpp

```diff
@@ -1,36 +1,89 @@
 // Copyright 2020-2020 the openage authors. See copying.md for legal info.
 
+#include "versions.h"
+
+#ifdef __linux__
 #include <gnu/libc-version.h>
+#endif
+
 #include <SDL2/SDL.h>
 #include <sstream>
+#include <eigen3/Eigen/Dense>
+#include <harfbuzz/hb.h>
+#include <QtGlobal>
+#include <epoxy/gl.h>
+#include <SDL2/SDL.h>
+#include <opus_defines.h>
 
-#include "versions.h"
+#include "versions/compiletime.h"
+
+#include "../util/strings.h"
+
+namespace openage::versions {
+
+std::map<std::string, std::string> get_version_numbers() {
 
-using namespace std;
+	std::map<std::string, std::string> version_numbers;
 
-namespace openage
-{
+	// Add SDL version number
+	SDL_version sdl_runtime_version;
+	SDL_GetVersion(&sdl_runtime_version);
+	version_numbers.emplace("SDL-runtime  ", util::sformat("%d.%d.%d",
```

Comment:
we shouldn't do the padding here, as this would add spaces to the generated dict keys.
the padding should rather be done in the printing routine, because only there it matters, the dict keys should contain the right values.